Jenkinsfile

```diff
@@ -0,0 +1,133 @@
+pipeline {
+
+    agent {
+        kubernetes {
+            defaultContainer 'maven'
+            yamlFile 'jenkins/agent.yaml'
+        }
+    }
+
+    environment {
+        GPG_SECRET     = credentials('presto-release-gpg-secret')
+        GPG_TRUST      = credentials("presto-release-gpg-trust")
+        GPG_PASSPHRASE = credentials("presto-release-gpg-passphrase")
+
+        GITHUB_OSS_TOKEN_ID = 'github-personal-token-wanglinsong'
+
+        SONATYPE_NEXUS_CREDS    = credentials('presto-sonatype-nexus-creds')
+        SONATYPE_NEXUS_PASSWORD = "$SONATYPE_NEXUS_CREDS_PSW"
+        SONATYPE_NEXUS_USERNAME = "$SONATYPE_NEXUS_CREDS_USR"
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '100'))
+        disableConcurrentBuilds()
+        disableResume()
+        overrideIndexTriggers(false)
+        timeout(time: 30, unit: 'MINUTES')
+        timestamps()
+    }
+
+    parameters {
+        booleanParam(name: 'PERFORM_MAVEN_RELEASE',
+                     defaultValue: false,
+                     description: 'Release and update development version when running in the master')
+    }
+
+    stages {
+        stage('Setup') {
+            steps {
+                sh 'apt update && apt install -y bash build-essential git gpg python3 python3-venv'
+            }
+        }
+
+        stage ('Prepare to Release') {
+            steps {
+                script {
+                    env.REPO_CURRENT_VERSION = sh(
+                        script: 'mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.version -q -ntp -DforceStdout',
+                        returnStdout: true).trim()
+                }
+                echo "current version: ${REPO_CURRENT_VERSION}"
+
+                sh '''
+                    git config --global --add safe.directory ${PWD}
+                    git config --global user.email "oss-release-bot@prestodb.io"
+                    git config --global user.name "oss-release-bot"
+                '''
+
+                sh '''#!/bin/bash -ex
+                    export GPG_TTY=$TTY
```

Comment:
It should be the following command instead:
```sh
export GPG_TTY=$(tty)
```
Original command works on Mac, but not on Linux, so in Jenkins logs I found that `GPG_TTY` value is empty:
```sh
[2024-06-12T09:28:00.089Z] + export GPG_TTY=
[2024-06-12T09:28:00.089Z] + GPG_TTY=
[2024-06-12T09:28:00.089Z] + gpg --batch --import ****
```
Test on Mac - both work:
```sh
$ export TEST=$TTY
$ echo $TEST
/dev/ttys002

$ export TEST=$(tty)
$ echo $TEST        
/dev/ttys002
```

Test on Ubuntu - only suggested command works:
```sh
$ export TEST=$TTY
$ echo $TEST
<-(empty line here)

$ export TEST=$(tty)
$ echo $TEST        
/dev/pts/0
```

Reply:
It is the old maven-gpg-plugin, the version is inherited from parent pom.
I have made a fix.

https://ci.ahana.dev/blue/organizations/jenkins/oss-presto-release%2Fpresto-maven-plugin/detail/PR-15/11/pipeline